README.md

```diff
@@ -72,6 +72,14 @@ limitPerRun: 30
 #     - confirmed
 ```
 
+## How long will it take?
+
+The app runs on a scheduled basis and in batches in order to avoid hitting rate limit ceilings. 
+
+Ergo, even after you initially install the GitHub configuration and add the `stale.yml` file, you may not see it act immediately.
```

Comment:
nit: 'ergo' isn't very commonly used in English, and I'd love for this to be as easily readable as possible. Instead maybe:

> What this means it that, even after...